platform/uboot/bootscript.cpp

```diff
@@ -101,6 +111,14 @@ FUNC_BEGIN(bootcmd_start)
  cp.b \$dtb_start \$dtbaddr \$dtb_size &&
  fdt addr \$dtbaddr &&
 #endif
+
+#ifdef platform_rockchip
+ adtimg get dt --id=\$main_fdt_id dtb_start dtb_size main_fdt_index &&
+ cp.b \$dtb_start \$dtbaddr \$dtb_size &&
+ fdt addr \$dtbaddr &&
+ EXTENV(bootargs, " earlyprintk console=ttyS2,1500000n8")
```

Comment:
Please move this into BoardConfig.mk and enable only for debug builds (See other device as reference)